frontend/src/Components/ULIComponent.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+type ULIComponentProps = {
+    outerDivClassName?: string; //for applying classes other than 'tooltip'
+    innerDivClassName?: string; //for applying classes other than 'w-4 h-4'
```

Comment:
Good thinking. Are there examples of other classes we need to apply to the outer div besides tooltip? 

Also, do we have another name besides `outerDiv` and `innerDiv`? I'm stuck on naming it but I think it should convey what it's doing and not what element it applies to (in case the element changes later on).

Reply:
This was one example, but  Carolina mentioned in the discussion of the refactor where there would be cases where there would be other classes applied to the component... specfically mentioning a case where there would be a need to override default width on the logo.

![image](https://github.com/user-attachments/assets/7e1e18c6-3320-4d6f-a783-0ad5bc6e9dc5)

@jtucholski I am always open for name suggestions.... always have the most difficult time with coming up something meaingful... :)

Reply:
How about we call it `toolTipClassName` and `iconClassName`?